tool/CMakeLists.txt

```
@@ -42,6 +42,9 @@ else()
   endif()
 endif()
 
+install(TARGETS bssl
+        RUNTIME DESTINATION ${CMAKE_INSTALL_LIBDIR})
```

Our top level CMake script defines CMAKE_INSTALL_BINDIR or it includes GNUInstallDirs which also defines it. 
```suggestion
        RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR})
```